src/loaders/DesktopArticleVideoLoader.tsx

```diff
@@ -1,10 +1,15 @@
 import React, { useEffect, useState } from 'react';
 import { DesktopArticleVideoLoaderProps } from 'loaders/types';
 import { setVersionWindowVar } from 'loaders/utils/GetVersion';
+import { shouldLoadUcpPlayer } from 'loaders/utils/shouldLoadPlayer';
 
 export { getVideoPlayerVersion } from 'loaders/utils/GetVersion';
 
 export const DesktopArticleVideoLoader: React.FC<DesktopArticleVideoLoaderProps> = ({ videoDetails }) => {
+	if (!shouldLoadUcpPlayer()) {
+		return;
+	}
+
```

Comment:
Quick question, can one do that? 

Won't it have an issue with `hooks` cannot be called conditionally ?

Reply:
I think the hooks will get called either way, but I'd move this below those, just above the other returns

Reply:
Actually inside the effect hook would probably be best, prevent it from getting run altogether

Reply:
Maybe something like

if (!player && shouldLoadUcpPlayer()) {
	getPlayer();
}

Reply:
I'll make the change. There's a deploy running on sandbox-s8 right now. I'll verify it works then, then redeploy with the above change you both discussed.

Reply:
@gundams-and-dragons @AlburIvan Fixed in 4398dd619dc73b31e4c4b49a8e1ef58f2f0e5a85

Reply:
@gundams-and-dragons is `canPlayVideo` created from the php side too? If not the hook could be an issue if our code somehow runs first.

Reply:
Just checked warrens slack. LGTM